CIScripts/Common/Invoke-CommandWithFunctions.ps1

```diff
@@ -0,0 +1,51 @@
+. $PSScriptRoot/Aliases.ps1
+function Invoke-CommandWithFunctions {
+    <#
+    .SYNOPSIS
+    This is a helper function for using locally defined functions in remote session.
+    The problem is that PowerShell does not support passing functions
+    to remote session, like ScriptBlocks or locally defined variables (by "$using:")
+    What Invoke-CommandWithFunctions do is just taking functions names and bodies 
+    and define them at remote session by using Invoke-Expression.
+    Then we invoke ScriptBlock with calls to already defined functions
```

Comment:
I don't think we should describe implementation of how the function does this, as it can change and someone could forget to update the documentation.

Let's just describe the behaviour of the function.